trinity/plugins/registry.py

```diff
@@ -8,5 +16,8 @@
 # config file which plugin is enabled or not
 
 ENABLED_PLUGINS = [
-    TxPlugin()
+    # Importing IPython (even just to check if it is available) breaks pytest.
+    # We detect if we are in a pytest session and just hardcoded the plugin then.
+    AttachPlugin() if "pytest" in sys.modules else get_suited_attach_plugin(),
```

Comment:
I'd love to drop this `if "pytest" in sys.modules` in favor of the `pkg_resources` approach suggested above.